llvm/include/llvm/IR/Attributes.td

```diff
@@ -303,6 +303,10 @@ def SanitizeNumericalStability : EnumAttr<"sanitize_numerical_stability", [FnAtt
 /// RealtimeSanitizer is on.
 def SanitizeRealtime : EnumAttr<"sanitize_realtime", [FnAttr]>;
 
+/// RealtimeSanitizer should error if an realtime_unsafe function is called
+/// during a sanitize_realtime function.
```

Comment:
During a sanitize_realtime function, or invoked below one

Perhaps something along the lines of:

"RealtimeSanitizer should error if invoked in a realtime context" -- idk documentation is hard.

Reply:
Done